provoke/worker/__init__.py

```diff
@@ -439,7 +439,8 @@ def _start_worker(self, worker):
             _current_worker_data = self._worker_data.copy()
             _current_worker_app = worker.app
             _run_cb(self, 'process_callback')
-            _run_cb(worker, 'process_callback')
+            _run_cb(self, 'start_callback')
```

Comment:
Remove this one, `start_callback` on the WorkerMaster object (self) is intended to be called in the master process and is called on [line 358](https://github.com/icgood/provoke/blob/master/provoke/worker/__init__.py#L358)